.github/CODEOWNERS

```diff
@@ -11,10 +11,10 @@
 # pull request.
 *       @hashicorp/cdktf
 
-# Laura likes to get notifications if changes happen to docs.
-# We need to include the CDKTF team as well to ensure Laura
-# is not the only one notified.
-website/docs/ @hashicorp/cdktf @laurapacilio @mgarrell777
+# Education likes to get notifications if changes happen to docs.
+# We need to include the CDKTF team as well to ensure Education
+# members are not the only ones notified.
+website/docs/ @hashicorp/cdktf @mgarrell777 @robin-norwood
```

Comment:
fyi: I just gave Robin write access to make this config change valid.
(We have the docs team with write access, that's why Laura and Matthew were valid without individual write access)